test/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM centos:centos7
+
+LABEL name="EL7 OSG 3.5 Gratia Probe Test Container"
+
+RUN yum install -y yum-priorities \
+    && yum install -y \
+       http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+    && yum install -y \
+       http://repo.opensciencegrid.org/osg/3.5/osg-3.5-el7-release-latest.rpm \
+    && yum install -y --enablerepo=devops osg-build \
+    && yum install -y --enablerepo=osg-empty slurm
+
+RUN cd \
+    && git clone --depth=1 https://github.com/opensciencegrid/gratia-probe \
+    && ver=$(awk '/Version:/ {print $2}' gratia-probe/rpm/gratia-probe.spec) \
+    && yum-builddep -y gratia-probe/rpm/gratia-probe.spec \
+    && mkdir -p bld/upstream \
+    && echo type=github repo=opensciencegrid/gratia-probe tag=master \
+            tarball=gratia-probe-$ver.tar.gz > bld/upstream/gp.source \
+    && osg-build rpmbuild bld \
+    && yum install -y ./bld/_build_results/gratia-probe-*-*-*.rpm
```

Comment:
Let's do separate `RUN`s for each of these.

Reply:
Ok... All of them?  :)

I mean i split it up this way because changes to the probes pretty much imply the second half of the steps need to be done also.  So i am curious if there will be a benefit to the extra layers

Reply:
Yeah, I think it makes it a little clearer if a particular command fails

Reply:
Ok, you got it.